002-quarkus-all-extensions/pom.xml

```diff
@@ -140,10 +140,6 @@
             <artifactId>woodstox-core</artifactId>
             <version>6.5.0</version>
         </dependency>
-        <dependency>
```

Comment:
Which error do you have at compile time with Neo4j dep?

Reply:
It can't find 999-SNAPSHOT for neo4j locally, I think it is my local (relocation) problem. I'll drop it and just remove exported otlp.

Reply:
You need to have enabled relocations module as neo4j moved to quarkiverse 

e.g. `mvn clean install -Dquickly -Prelocations  -Dno-test-modules`

Reply:
didn't know, thank you